components/service/sync-autofill/src/test/java/mozilla/components/service/sync/autofill/GeckoCreditCardsAddressesStorageDelegateTest.kt

```diff
@@ -74,15 +78,15 @@ class GeckoCreditCardsAddressesStorageDelegateTest {
 
     @Test
     fun `WHEN onAddressFetch is called THEN the storage is called to gett all addresses`() {
```

Comment:
This test is to be removed in https://github.com/mozilla-mobile/android-components/pull/12138 (replaced by tests for both situations where address autocomplete is enabled or disabled) so I prefer to keep this previous misspelling in order to avoid adding a soon to be an obsolete entry in the git history.